h/static/scripts/widget-controller.js

```diff
@@ -312,9 +347,11 @@ module.exports = function WidgetController(
     streamer.reconnect();
   });
 
-  // When a direct-linked annotation is successfully anchored in the page,
-  // focus and scroll to it
   $rootScope.$on(events.ANNOTATIONS_SYNCED, function (event, tags) {
```

Comment:
This is a bit confusing, we are handling the `ANNOTATIONS_SYNCED` event in two different places in this code and doing different things in each case - in one place it is aware of the different tabs and in the other place it isn't.